apps/testing/src/main/java/com/akto/testing/Main.java

```diff
@@ -60,30 +66,32 @@ private static ObjectId createTRRSummaryIfAbsent(TestingRun testingRun, int star
 
     private static void setupRateLimitWatcher () {
         scheduler.scheduleAtFixedRate(new Runnable() {
-            @Override
             public void run() {
-                Context.accountId.set(1_000_000);
-                AccountSettings settings = AccountSettingsDao.instance.findOne(AccountSettingsDao.generateFilter());
-                if (settings == null) {
-                    return;
-                }
-                int globalRateLimit = settings.getGlobalRateLimit();
-                Map<ApiRateLimit, Integer> rateLimitMap =  RateLimitHandler.getInstance().getRateLimitsMap();
-                rateLimitMap.clear();
-                rateLimitMap.put(new GlobalApiRateLimit(globalRateLimit), globalRateLimit);
+                AccountTask.instance.executeTask(new Consumer<Account>() {
```

Comment:
didn't get the working of this block of code. So basically this will iterate on all active accounts, and override  rateLimitMap